doc/Requirements Specification/Parts/Data.lyx

```diff
@@ -424,7 +424,20 @@ glsuserii{internal action}
 
 \end_inset
 
- execution time.
+ execution time and the CPU 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glspl{resource demand}
```

Comment:
in the PDF is written "resouce demandss" with double "s". I think it is defined as "resource demands" so use gls (or change definition to "resource demand".